media-video/guvcview/guvcview-2.2.1.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="Simple Qt6 or GTK+3 interface for capturing and viewing video from v4l2 devices"
+HOMEPAGE="https://guvcview.sourceforge.net/"
+SRC_URI="https://downloads.sourceforge.net/project/${PN}/source/${PN}-src-${PV}.tar.bz2"
+S="${WORKDIR}/${PN}-src-${PV}"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="gsl +gtk pulseaudio qt6 +sdl sfml"
+REQUIRED_USE="
+	|| ( gtk qt6 )
```

Comment:
It is certainly interesting that `IUSE="gtk qt6"` is actually possible and either can be picked at application start using GUI_API - however, it might come a bit unexpected to the user.

Reply:
For the user, you're probably right.